Modules/Messages/src/Messages.ConsumerApi/MessagesModule.cs

```diff
@@ -27,12 +27,18 @@ public override void ConfigureServices(IServiceCollection services, IConfigurati
             options.DbOptions.Provider = parsedConfiguration.Infrastructure.SqlDatabase.Provider;
             options.DbOptions.DbConnectionString = parsedConfiguration.Infrastructure.SqlDatabase.ConnectionString;
 
-            options.BlobStorageOptions.CloudProvider = parsedConfiguration.Infrastructure.BlobStorage.CloudProvider;
-            options.BlobStorageOptions.ConnectionInfo = parsedConfiguration.Infrastructure.BlobStorage.ConnectionInfo;
-            options.BlobStorageOptions.Container =
-                parsedConfiguration.Infrastructure.BlobStorage.ContainerName.IsNullOrEmpty()
-                    ? "messages"
-                    : parsedConfiguration.Infrastructure.BlobStorage.ContainerName;
+            if (parsedConfiguration.Infrastructure.BlobStorage != null)
+            {
+                options.BlobStorageOptions = new()
+                {
+                    CloudProvider = parsedConfiguration.Infrastructure.BlobStorage.CloudProvider,
+                    ConnectionInfo = parsedConfiguration.Infrastructure.BlobStorage.ConnectionInfo,
+                    Container =
+                    parsedConfiguration.Infrastructure.BlobStorage.ContainerName.IsNullOrEmpty()
+                        ? "messages"
+                        : parsedConfiguration.Infrastructure.BlobStorage.ContainerName
```

Comment:
Missing indentation

Reply:
Fixed by moving line 37 to previous line, if it doesn't work I will make it exactly as shown here.